avocado/utils/linux.py

```diff
@@ -72,3 +72,18 @@ def enable_selinux_enforcing():
     if is_selinux_enforcing():
         return True
     return False
+
+
+def is_os_secureboot_enabled(self):
+    """
+    Check whether the secure-boot is enabled at os level.
+    Check for "00000002" in "/proc/device-tree/ibm,secure-boot" file
+    If found, then secure-boot is enabled.
+
+    :return: True if secureboot is enabled, False if otherwise
+    """
+    cmd = "lsprop /proc/device-tree/ibm,secure-boot"
```

Comment:
Hi @vaishnavibhat, can you please describe to me what `lsprop` is and what it does, to be hones I am not familiar with it.